Library/Homebrew/cmd/install.rb

```diff
@@ -302,6 +306,51 @@ def run
         Install.perform_preinstall_checks_once
         Install.check_cc_argv(args.cc)
 
+        # Showing dependencies and required size to install
+        if args.ask? || Homebrew::EnvConfig.ask?
+          ohai "Looking for dependencies..."
+          package = []
+          bottle_size = 0
+          installed_size = 0
+          installed_formulae.each do |f|
+            next unless (bottle = f.bottle)
+
+            begin
+              bottle.fetch_tab(quiet: !args.debug?)
```

Comment:
```suggestion
              bottle.fetch_tab
```
feels like it may be worth communicating that this is downloading stuff?

Reply:
That's a good idea, I was following `cmd/info.rb` file but it wasn't maybe the best idea

Reply:
I kept the quiet arguments since this will fetch many JSON files—one for each dependency. Without it, the terminal would be flooded with output and wouldn’t be as readable.